Training/Training/file.c

```diff
@@ -1,13 +1,85 @@
-// ------------------------------------------------------------------------------------------------
+﻿// ------------------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
-// Copyright (c) Metamation India.
+// Copyright Gokul TS (c) Metamation India.
 // ------------------------------------------------------------------
-// Program.c
+// file.c
 // Program on main branch.
 // ------------------------------------------------------------------------------------------------
-
 #include <stdio.h>
+#include <fcntl.h>
+#include <io.h>
+
+#define ANSI_RESET_ALL "\x1b[0m"
+#define ANSI_COLOR_RED "\x1b[31m"
+#define ANSI_COLOR_GREEN "\x1b[32m"
+#define ANSI_COLOR_YELLOW "\x1b[33m"
+#define ROW 8
+#define COLUMN 8
+
+/// <summary>This function prints the input into both console and text file</summary>
+void Print (const wchar_t* const fText, FILE* file);
+/// <summary>Returns whether the two input files are equal or not</summary>
+int Compare_Files (const char* file1, const char* file2);
+
+int main () {
+   int prevMode = _setmode (_fileno (stdout), _O_U8TEXT);
+   FILE* file = fopen ("test_file.txt", "w, ccs=UTF-8");
```

Comment:
Try to give a comment while using characters like these. Apply this change wherever applicable.